packages/manager/modules/vps/src/dashboard/migrate/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,12 @@
+{
+  "vps_dashboard_migrate_configuration_vcore": "{{count}} vCores",
+  "vps_dashboard_migrate_configuration_label_ram": "Mémoire vive :",
+  "vps_dashboard_migrate_configuration_label_core": "Processeur :",
+  "vps_dashboard_migrate_configuration_label_disk": "Stockage :",
+  "vps_dashboard_migrate_title": "Migration de votre VPS",
+  "vps_dashboard_migrate_faq_title": "Pour plus de détails, ",
+  "vps_dashboard_migrate_faq_link_text": "veuillez consulter la FAQ dédiée.",
```

Comment:
The whole sentence should be the link for simplicity

Reply:
Will check with PU for this change.